src/index.ts

```diff
@@ -854,6 +863,88 @@ export abstract class PinejsClientCore<PinejsClient> {
 		}
 	}
 
+	private waitAsync(delayMs: number): Promise<void> {
+		return new Promise<void>((resolve) => {
+			setTimeout(resolve, delayMs);
+		});
+	}
+
+	protected async callWithDelay<T>(
+		fnCall: (attempts: number, isLastAttempt: boolean) => Promise<T>,
```

Comment:
This can be changed to
```suggestion
		fnCall: () => Promise<T>,
```
since we don't actually care for those parameters in our usage (and since it's a protected method it's hard for anything else to do so)